### PR TITLE
Merge dev into main (Release 0.16.0)

### PR DESCRIPTION
**Pull request by Pronyo-Chan**

### CODE REVIEW

File: maps/graph_edges/base_graph/edges_dgb.py [hunk 1]
```diff
@@ -55,43 +55,43 @@
...
     {"from": {"map": "DGB_06", "id": 1}, "to": {"map": "DGB_06", "id": 0}, "reqs": []}, #? Basement Fall From Ceiling -> Basement Door Bottom Left
-    
+
     {"from": {"map": "DGB_06", "id": 1},        "to": {"map": "DGB_06", "id": "ChestA"}, "reqs": []}, #* Basement Fall From Ceiling -> ChestA (CastleKey1)
     {"from": {"map": "DGB_06", "id": "ChestA"}, "to": {"map": "DGB_06", "id": 1},        "reqs": []}, #* ChestA (CastleKey1) -> Basement Fall From Ceiling
 
     # DGB_07 Study (1F)
     {"from": {"map": "DGB_07", "id": 0}, "to": {"map": "DGB_02", "id": 2}, "reqs": []}, # Study (1F) Door Bottom -> West Hall (1F) Door Top Left
-    
+# TODO: table reachable with laki jump
```
Comment (Pronyo-Chan) [inline maps/graph_edges/base_graph/edges_dgb.py:87]: Can this be removed? It should be handled properly in the glitch edge file

**Comment by Pronyo-Chan** [comment]
lgtm, updated the website with the new tubba table trick on dev.

**Comment by icebound777** [comment]
lgtm after Pronyo checks my latest [commit](https://github.com/icebound777/PMR-SeedGenerator/pull/56/commits/3276ee8c207bf9738bbbf6dedfe5e217233e661b)

**Comment by Pronyo-Chan** [comment]
lgtm

**Pull request merged**